docs/examples/crawler_login.mdx

```diff
@@ -0,0 +1,30 @@
+---
+id: crawler-login
+title: Crawler Login
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+import RunnableCodeBlock from '@site/src/components/RunnableCodeBlock';
+
+import PlaywrightLogin from '!!raw-loader!roa-loader!./code_examples/playwright_login.py';
+import HttpLogin from '!!raw-loader!roa-loader!./code_examples/http_login.py';
+
+This example demonstrates how to implement login functionality with <ApiLink to="class/PlaywrightCrawler">`PlaywrightCrawler`</ApiLink> and <ApiLink to="class/HttpCrawler">`HttpCrawler`</ApiLink>.
+
+When you need to authenticate on a website, you typically want to use only one <ApiLink to="class/Session">`Session`</ApiLink> throughout the crawl. This requires additional configuration of the <ApiLink to="class/SessionPool">`SessionPool`</ApiLink>. Learn more in the [session management guide](../guides/session-management).
+
```

Comment:
Maybe we should also give a hint on how to have multiple sessions with different credentials, for example? Like, just a single sentence about new_session_function probably.